pulp_container/app/modelresource.py

```diff
@@ -1,7 +1,46 @@
 from import_export import fields, widgets
 from pulpcore.plugin.importexport import QueryModelResource, BaseContentResource
+from pulpcore.plugin.modelresources import RepositoryResource
 
-from pulp_container.app.models import Blob, Manifest, ManifestListManifest, ManifestSignature, Tag
+from pulp_container.app.models import (
+    Blob,
+    Manifest,
+    ManifestListManifest,
+    ManifestSignature,
+    Tag,
+    ContainerRepository,
+    ContainerPushRepository,
+)
+
+
+class ContainerRepositoryResource(RepositoryResource):
+    """
+    A resource for importing/exporting repositories of the pull/sync type
+    """
+
+    def set_up_queryset(self):
+        """
+        :return: A queryset containing one sync repository that will be exported.
+        """
+        return ContainerRepository.objects.filter(pk=self.repo_version.repository)
+
+    class Meta:
+        model = ContainerRepository
```

Comment:
what about import_id_fields?

Reply:
exclude fields too?

Reply:
`exclude_fields` is not required to declare. Same for `import_id_fields` ("The default field for object identification is id, you can optionally set which fields are used as the id when importing").

Reply:
Actually, I should exclude "manifest_signing_service" from the fields. Does it make sense to include signing services in the export? I believe, it does not.